playground/mocks/data/idp/idx/enroll-profile-with-password.json

```diff
@@ -102,17 +102,17 @@
       ],
       "settings": {
         "complexity": {
-          "minLength": "NUMBER",
-          "minLowerCase": "NUMBER",
-          "minUpperCase": "NUMBER",
-          "minNumber": "NUMBER",
-          "minSymbol": "NUMBER",
-          "excludeUsername": "BOOLEAN",
```

Comment:
These values for this mock file are invalid, looked at others and they all have proper values (numeric or boolean). In v3, without updating this the requirements wont display.